topics/application-gradle-plugin.md

```diff
@@ -0,0 +1,68 @@
+[//]: # (title: Distribution via Gradle Application plugin)
+
+From the Gradle [documentation](https://docs.gradle.org/current/userguide/application_plugin.html):
+> The Application plugin facilitates creating an executable JVM application. 
+> It makes it easy to start the application locally during development, 
+> and to package the application as a TAR and/or ZIP including operating system specific start scripts.
+
+We will use the second responsibility of this plugin and show how to package a simple Ktor application.
```

Comment:
not sure we need to explicitly say "second responsibility"

Reply:
+1